test/src/xquery/fn.xql

```diff
@@ -58,8 +58,17 @@ function fnt:doc-available($filename as xs:string) {
 
 declare
     %test:assertEquals(0, 0)
-function fnt:tokenize-empty($filename as xs:string) {
+function fnt:tokenize-empty() {
      count(() => tokenize("\s")),
      tokenize((), "\s")
 };
```

Comment:
This test is failing with:

> assertEquals failed: wrong number of items returned by function. Expected: 2. Got: 1. Expected: '0 0' Actual: '0'

Perhaps this test is failing because the `count(...)` expression returns 0, while the `tokenize(...)` expression returns an empty sequence. Consider changing the latter to `count(tokenize(...))` or `tokenize(...) => count()`